update-afc.sh

```
@@ -61,7 +61,7 @@ main() {
   afc_config_dir="${printer_config_dir}/AFC"
   afc_file="${afc_config_dir}/AFC.cfg"
   moonraker_config_file="${moonraker_config_file:-${printer_config_dir}/moonraker.conf}"
-  afc_path="$HOME/AFC-Klipper-Add-On"
+  afc_path="${SCRIPT_DIR}"
```

Shellcheck (0.11.0)

[warning] afc_path appears unused. Verify use (or export if used externally).

(SC2034)